documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -58,246 +86,90 @@ base-char = 1-9 A-H J-N P-Z a-k m-z
-```json
-"proof": {
-    "type": "JcsEd25519Signature2020",
-    "verificationMethod": "did:iota:GzXeqBXGCbuebiFtKo4JDNo6CmYmGbqxyh2fDVKadiBG#authentication",
-    "signatureValue": "3fLtv3KUU4T5bHNLprV3UQ2Te3bcRZ9uUYSFouEA7fmYthieV35NNLqbKUu8t2QmzYgnfp1KMzCqPzGNi3RjU822"
-}
 ```
 
-### Anatomy of the Diff object
+## Controllers
 
-The `diff` object MUST contain all the differences between the current and previous DID Document and its DID Document Metadata. The differentiation is formatted as a JSON object that includes the differences between the two DID Document objects and the two DID Document Metadata objects. Exact details of how this is generated will be added later.
+As of now, only one state and one governance controller can be set for an Alias Output. Multiple controller support can be done by future updates.
 
-Example `diff` of adding a new service entry to the document and changing the `updated` field in the metadata:
-```json
-{
-  "diff": {
-    "doc": {
-      "service": [
-        {
-          "id": "did:iota:X7U84ez4YeaLwpfdnhdgFyPLa53twvAuMSYdRQas54e#linked-domain-1",
-          "type_": "LinkedDomains",
-          "service_endpoint": "https://example.com/"
-        }
-      ]
-    },
-    "meta": {
-      "updated": "2022-01-21T09:50:28Z"
-    }
-  }
-}
-```
+A state controller can directly update the DID Document and the amount of IOTA coins held by the Output, but it can't destroy the output. A governor controller, one the other hand, can indirectly update the DID Document by updating the state controller. It also can destroy the output by having a governance transition to the empty state.
 
 ## CRUD Operations
 
-Create, Read, Update and Delete (CRUD) operations that change the DID Documents are to be submitted to an IOTA Tangle in order to be publicly available. They will either have to be a valid Int DID message or Diff DID message, submitted on the correct index of the identity. 
+Create, Read, Update and Delete (CRUD) operations that change the DID Documents are to be submitted to an IOTA Tangle in order to be publicly available.
+They have to update or destroy the Alias Output including the DID document.
 
 ### Create
 
-To generate a new DID, the method described in [generation](#generation) must be followed. A basic DID Document must be created that includes the public key used in the DID creation process as a `verificationMethod` with a capability invocation verification relationship. This DID Document must be formatted as an Integration DID message, signed using the same keypair used to generate the tag, and published to an IOTA Tangle on the index generated out of the public key used in the DID creation process. 
+Creating a new DID is done by creating a new Alias Output containing a serialized document as described in [Anatomy of serialized document](#anatomy-of-serialized-document). This requires the fulfillment of all the
+Stardust requirements including the byte cost.
 
 ### Read
```

Comment:
This needs to detail how to obtain the Alias Id from a DID, unpack the state metadata and, for instance, replace the placeholder `did:0:0` with the actual DID.